lambda/custom/document/renderPage.json

```diff
@@ -0,0 +1,138 @@
+{
+    "type": "APL",
+    "version": "1.0",
+    "settings": {},
+    "theme": "dark",
+    "import": [
+            { 
+                "name" : "alexa-styles", 
+                "version" : "1.0.0" 
+            } 
+        ],
+    "resources": [
+        { 
+            "colors" : {
+               "SkillBackgroundColor" : "blue" , 
+               "SkillHighlightTextColor"  : "yellow" , 
+               "SkillTextColor" : "white"
+            }
+        } 
+        
+        ],
+    "styles": {
+        
+        
+    },
+    "onMount": [],
+    "graphics": {},
+    "commands": {},
+    "layouts": {
+        "CenteredText" : 
+        { 
+            "parameters" :  [
+                "text", 
+                "speech", 
+                "cid" 
```

Comment:
I'm pretty sure you do not need this and can set the id property directly on the layout when it is used. I have done this with containers in a layout, for instance.

Reply:
Not sure why this is an issue.